### PR TITLE
Spooky Nukie names for Halloween Mode

### PR DESCRIPTION
**Pull request by Gannets**
## About the PR 
Like the festive name list added for Spacemas, this PR adds some gruesome, spooky names to the nuclear operative name list when Halloween is defined.

`#ifdef HALLOWEEN
	name += pick("Hell", "Demon", "Blood", "Murder", "Gore", "Grave", "Sin", "Slaughter")`

## Why's this needed? 
Low effort spooky addition 👻

### CODE REVIEW

**Review by ZeWaka — COMMENTED** [review]
Can't have two #elses

**Comment by Gannets** [comment]
Weird, wonder how I got that working local. What's a good solution for two defines like this?

**Comment by ZeWaka** [comment]
it compiles, but doesn't do what you actually want it to do

File: code/datums/gamemodes/nuclear.dm [hunk 1]
```diff
@@ -436,6 +436,9 @@ var/syndicate_name = null
 	// Prefix
 #ifdef XMAS
 	name += pick("Merry", "Jingle", "Holiday", "Santa", "Gift", "Elf", "Jolly")
+#else
+#ifdef HALLOWEEN
+	name += pick("Hell", "Demon", "Blood", "Murder", "Gore", "Grave", "Sin", "Slaughter")
 #else
```
Comment (ZeWaka) [inline code/datums/gamemodes/nuclear.dm:442]:
```suggestion
#if defined(XMAS)
	name += pick("Merry", "Jingle", "Holiday", "Santa", "Gift", "Elf", "Jolly")
#elif defined(HALLOWEEN)
	name += pick("Hell", "Demon", "Blood", "Murder", "Gore", "Grave", "Sin", "Slaughter")
#else
```
:)

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Pull request merged**